CMakeLists.txt

```diff
@@ -131,6 +131,7 @@ check_function_exists (sigaltstack HAVE_SIGALTSTACK)
 check_cxx_symbol_exists (backtrace execinfo.h HAVE_EXECINFO_BACKTRACE)
 check_cxx_symbol_exists (backtrace_symbols execinfo.h
   HAVE_EXECINFO_BACKTRACE_SYMBOLS)
+check_cxx_symbol_exists (_chsize_s io.h HAVE__CHSIZE_S)
```

Comment:
Thanks for the update!

One thing that is still missing is a `#cmakedefine` entry for `HAVE__CHSIZE_S` in `src/config.h.cmake.in` similar to this use:

https://github.com/google/glog/blob/5d4fa3d7742ed89f975730c73250bbe1d3ab1521/src/config.h.cmake.in#L130-L131

Reply:
hi @sergiud , I have already updated it